index.html

```diff
@@ -27,6 +27,29 @@
         </nav>
       </header>
 
+        <div id="mobile-menu-modal" class="toogleModalDisplay">
+          <div id="mobile-menu">
+            <div id="modal-close-icon">
+              <img
+                src="./icons/modal-close-icon.png"
+                alt="close-mobile-menu"
+                width="20rem"
+              />
```

Comment:
- [ ] The thing with this section is, I have tried to find out where has gone the hamburger icon but I could not, before I start testing the app, please make sure you fix this design before we jump on the next steps as you go, I just remark that the path to the image is not correct, you are missing /images/... in your path to the image, either it is not the hamburger image but at least it will help.

<img width="519" alt="1" src="https://user-images.githubusercontent.com/77843050/215571437-b3f053cb-1eaf-4f5f-b1b1-5cf41c1148bc.png">

Reply:
Hello [@joskalenda ], thanks for the reply. But the hamburger button is there. I have included screenshots of different sizes.
![image](https://user-images.githubusercontent.com/100337086/215667535-e6cbacf2-78a5-4196-abb3-7bf631e94acf.png)
![image](https://user-images.githubusercontent.com/100337086/215667845-14fcddaa-2168-4cc0-9244-30963ab4b3c7.png)